grafana-plugin/src/utils/consts.ts

```diff
@@ -2,7 +2,7 @@ import plugin from '../../package.json'; // eslint-disable-line
 
 // Navbar
 export const APP_TITLE = 'Grafana OnCall';
-export const APP_SUBTITLE = `Developer-friendly incident response (${plugin?.version})`;
+export const APP_SUBTITLE = `Developer-friendly alert groups response (${plugin?.version})`;
```

Comment:
hmm, imo "alert groups" response doesn't fully summarize what our app does. From the perspective of someone who is new to the app, it might not be entirely clear what an "alert group" is. I would vote for keeping the word `incident` here.

whatever terminology we decide on, we should make sure we're consistent throughout the codebase. We have this same statement in a few places, here, in the helm chart, and the main README (maybe some other spots too?). 

If we do decide to go w/ "alert groups" I would suggest changing it to the singular version:
```suggestion
export const APP_SUBTITLE = `Developer-friendly alert group response (${plugin?.version})`;
```

Reply:
Yes, I agree, here we should leave 'incident' word